packages/calcite-components/src/components/shell-panel/shell-panel.scss

```diff
@@ -179,97 +154,160 @@
   @apply flex flex-auto overflow-hidden;
 }
 
-.content {
-  @apply bg-background
-  pointer-events-auto
-  flex
-  flex-col
-  flex-nowrap
-  items-stretch
-  self-stretch
-  p-0;
-  transition:
-    max-block-size var(--calcite-animation-timing),
-    max-inline-size var(--calcite-animation-timing);
-}
+:host([layout="vertical"]) {
+  --calcite-internal-shell-panel-z-index: calc(theme("zIndex.default") + 1);
 
-:host([layout="vertical"]) .content {
-  inline-size: var(--calcite-shell-panel-width-internal);
-  max-inline-size: var(--calcite-shell-panel-max-width-internal);
-  min-inline-size: var(--calcite-shell-panel-min-width-internal);
-}
+  .content {
+    inline-size: var(--calcite-shell-panel-width, var(--calcite-internal-shell-panel-width));
```

Comment:
Should `--calcite-shell-panel-width` be renamed to `--calcite-shell-panel-content-width`?